packages/rest/test/unit/request-body.validator.test.ts

```diff
@@ -201,13 +299,18 @@ describe('validateRequestBody', () => {
 
 function verifyValidationRejectsInputWithError(
   errorMatcher: Error | RegExp | string,
+  details: RestHttpErrors.ErrorDetails[] | undefined,
   body: object | null,
   spec: RequestBodyObject | undefined,
   schemas?: SchemasObject,
 ) {
-  // workaround for Function.prototype.bind not preserving argument types
-  function validateRequestBodyWithBoundArgs() {
+  try {
     validateRequestBody(body, spec, schemas);
+    throw new Error(
```

Comment:
We should wrap this error in a variable and assert this in the catch block: `expect(err).to.not.eql(noErrorThrown)`

Reply:
@shimks that error doesn't have the same `error.message` and `error.details` as the `catch` block expect to have, so if it happens the 1st assertion ` expect(err.message).to.equal(errorMatcher);` will faill :)

Reply:
Although I think it's cleaner to directly assert for the error instance created at the moment the validation (unexpectedly) passes, you're right in the sense that the error thrown by AJV is extremely unlikely to have `errorMatcher` as its message.

Feel free to ignore this comment